eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -172,13 +188,24 @@ jobs:
               targetFolder: $(Build.ArtifactStagingDirectory)/docs/${{ artifact.Name }}
             displayName: Copy documentation to artifact staging directory
 
+          - task: CopyFiles@2
+            inputs:
+              sourceFolder: build/sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Path }}/docs/docs.ms
+              targetFolder: $(Build.ArtifactStagingDirectory)/docs.ms/${{ artifact.Name }}
+            displayName: Copy docs.microsoft.com markdown to artifact staging directory
+
       - task: PublishPipelineArtifact@1
         inputs:
           artifactName: packages
           path: $(Build.ArtifactStagingDirectory)/packages
 
-      # After all docs artifacts are generated publis docs artifacts
+      # After all docs artifacts are generated publish docs artifacts
       - task: PublishPipelineArtifact@1
         inputs:
           artifactName: docs
           path: $(Build.ArtifactStagingDirectory)/docs
+
+      - task: PublishPipelineArtifact@1
```

Comment:
Is there a reason we are creating a new artifact for these docs? Instead of just putting them the packages artifact grouped together with the artifacts? 

I've been trying to get our artifacts all together in one artifact grouped into folders by the package. 

cc @scbedd

Reply:
The choice to separate them into a different folder is mostly arbitrary and because they aren't useful to downstream publishing (yet). Docs have tended to be separate artifacts but that decision was arbitrary as well... One possible future state could be: 

```
- packages
  - azure-core
    - package.json
    - CHANGELOG.md
    - docs
      - docs.ms
        - api-docs.md
      - api
        - <doxygen files> 
```

Changes to this would require additional downstream changes but I think this is a more coherent approach.

Reply:
Filed: https://github.com/Azure/azure-sdk-for-cpp/issues/587 as an engsys improvement